### DISCUSSION TITLE
Enroll Response

### DISCUSSION THREAD

**Discussion by King-anderson** [Koenkk/zigbee2mqtt#12192] [General]
Hi There, 
I want to investigate how the Enroll Response is handled in Zigbee2MQTT. Can anyone help me?

**Comment by Koenkk** [comment]
It is handled here: https://github.com/Koenkk/zigbee-herdsman/blob/f4f7b1b3df3a423dc8010e0ed830d0d72d5afa3a/src/controller/model/device.ts#L213
